react/Pill/Pill.js

```diff
@@ -9,7 +10,7 @@ import CrossIcon from '../CrossIcon/CrossIcon';
 import ScreenReaderOnly from '../ScreenReaderOnly/ScreenReaderOnly';
 
 type Props = {
-  text: string,
+  text: PropTypes.node,
```

Comment:
This component is using flow type definitions that compile to `PropTypes` rather than setting prop types directly. In this case we should use `React$Element` here.

Reply:
Actually look at the [usage below](https://github.com/seek-oss/seek-style-guide/pull/509/files#diff-e73fc4fe5c618a913a9593e8926bf867R34) this api should probably not be changed as the `text` prop is rendered in a `Text` component which should not be the case for arbitrary children. We should probably have a quick chat about how best to accommodate this

Reply:
Ignore me, use `React$Node` to support both strings and node https://github.com/facebook/flow/blob/master/lib/react.js#L12